src/intent.html

```diff
@@ -87,23 +87,32 @@ <h3 id="mixing_intent_grammar">The Grammar for <code class="attribue">intent</co
       application specific function or concept.
       For many concepts, the words used to speak a concept are very similar to the name
       used when referencing a concept.
-      A [=concept=] matches a name in an [=Intent Concept Dictionary=] recognized by the AT,
-      to produce specific audio or braille renderings based on the speech hints
-      given in the dictionary.</p>
+      A <dfn id="intent_known_concept">known concept</dfn> matches a name
+      in an [=Intent Concept Dictionary=] recognized by the AT.
+      This may produce specific audio or braille renderings based on the speech hints
+      given in the dictionary.
+      An <dfn id="intent_unknown_concept">unknown concept</dfn> is a concept not
+      currently known to the AT. These will be treated the same as a [=literal=], spoken as-is.
+      However, future updates of the AT and [=Intent Concept Dictionary=] may
+      include additional concepts, at which time those concepts may also receive special treatment.</p>
     </li>
     <li>
-      <p>A <dfn id="intent_literal">literal</dfn> is a name that does not match a [=concept=]
-      name known to the application.  In this case, a default reading is generated by replacing any
+      <p>A <dfn id="intent_literal">literal</dfn> is a name starting with <q>`_`</q> (U+00F5).
+      These will never be included in an [=Intent Concept Dictionary=].
+      The reading of a literal is generated by replacing any
       <code>-</code>, <code>_</code>, <code>.</code> in the name by
       spaces and then reading the resulting phrase.
-      Names starting with <q>`_`</q> (U+00F5)  are always considered to be [=literal=] names,
-      and should never be in an [=Intent Concept Dictionary=].</p>
+      Note that while AT will often add "glue" words when generating speech
+      (such as the "of" when <code>intent="f(x)"<code> generates "f of x"),
+      an empty literal "_" generates no glue words;
+      effectively it implies the <code>:silent</code>
```

Comment:
Actually, upon rereading, I think it's better to leave this out to keep the initial itemization concise.

OTOH, I think that the default fixity discussion should probably move "up" (either before our after the concept/literal `<ul>`), since it is not specific to literals but applies to concepts as well.  Perhaps it's worth making explicit that `:silent` should also omit any glue words.

Reply:
OK as that has gone in last commit, OK to approve this PR.

(re-)ordering of text likely to come up anyway now intent split to its own chapter and some possible merges of text from accessbility appendix.